_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,304 @@
+
+		<p>
+			Open this slider on a device with a motion sensor, such as a smart phone or tablet. Rotate the device to adjust
+			the slider value. The check box disables the motion sensing adjustment.
+		</p>
+		<p>Note: This example may not work across all browsers.</p>
+
+		<div>
+			<input type="range" min="1" max="100" value="50" id="motionSlider" disabled />
+			<p aria-live="polite">Slider Value: <span id="output">50</span></p>
+		</div>
+		<div>
+			<input type="checkbox" id="disableMotion" />
+			<label for="disableMotion">Disable Motion Actuation</label>
+		</div>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This [HTML document][] that can be operated through the device's orientation to increase and decrease the value of a slider but has no way to disable this functionality.
+
+```html
+<html>
+	<head>
+		<title>Failed Example 1</title>
+		<script src="/test-assets/7677a9/slider.js"></script>
+		<script>
+			function activateSlider() {
+				window.addEventListener('deviceorientation', handleOrientation)
+			}
+		</script>
+	</head>
+
+	<body onload="activateSlider();">
+		<pre class="output"></pre>
+
+		<h1>Slider Motion Sensor Example</h1>
+
+		<p>
+			Open this slider on a device with a motion sensor, such as a smartphone or tablet. Tilt the device to the right
+			and left to adjust the slider value.
+		</p>
+		<p>Note: This example may not work across all browsers.</p>
+
+		<div>
+			<input type="range" min="1" max="100" value="50" id="motionSlider" disabled />
+			<button id="increaseSlider" type="button">Increase Value</button>
+			<p aria-live="polite">Slider Value: <span id="output">50</span></p>
+		</div>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+This [HTML document][] that can be operated through the device's orientation to increase and decrease the value of a slider has a control to disable the functionality but it is not [included in the accessibility tree][].
+
+```html
+<html>
+	<head>
+		<title>Failed Example 2</title>
+		<script src="/test-assets/7677a9/slider.js"></script>
+		<script>
+			function activateSlider() {
+				window.addEventListener('deviceorientation', handleOrientationCanBeDisabled)
+			}
+		</script>
+	</head>
+
+	<body onload="activateSlider();">
+		<h1>Slider Motion Sensor Example</h1>
+
+		<p>
+			Open this slider on a device with a motion sensor, such as a smart phone or tablet. Tilt the device to the right
+			and left to adjust the slider value. The check box disables the motion sensing adjustment.
+		</p>
+		<p>Note: This example may not work across all browsers.</p>
+
+		<div>
+			<input type="range" min="1" max="100" value="50" id="motionSlider" disabled />
+			<p aria-live="polite">Slider Value: <span id="output">50</span></p>
+		</div>
+		<div aria-hidden="true">
+			<input type="checkbox" id="disableMotion" />
+			<label for="disableMotion">Disable Motion Actuation</label>
+		</div>
+	</body>
+</html>
+```
+
+#### Failed Example 3
```

Comment:
I'd move the "not visible" example before the "not accessible" one in order to mimic the order of the conditions in the expectations.